boards/arm/nrf52840dongle_nrf52840/Kconfig.defconfig

```diff
@@ -24,6 +24,42 @@ config FLASH_LOAD_OFFSET
 	default 0x1000
 	depends on BOARD_HAS_NRF5_BOOTLOADER && !USE_DT_CODE_PARTITION
 
+if BOARD_SERIAL_BACKEND_CDC_ACM
+
+config USB_DEVICE_STACK
+	default y
+
+config USB_CDC_ACM
+	default y
+
+config UART_CONSOLE
+	default CONSOLE
+
+config USB_DEVICE_INITIALIZE_AT_BOOT
+	default y if !MCUBOOT
+
+config SHELL_BACKEND_SERIAL_CHECK_DTR
+	default SHELL
+
+config USB_DEVICE_REMOTE_WAKEUP
+	default n
```

Comment:
More of a curiosity: why default this to false? The application only uses it if enabled, it's a cool demo to show that the driver has wakeup support.

Reply:
Have removed it, might need to reintroduce it if it causes problems (was introduced on thingy53 because if the application does not use the API then it can get into a state whereby the device locks up)

Reply:
Driver supports remote wakeup, but CDC ACM implementation does not seem to issue remote wakeup on its own. Therefore, if host suspends the device (Linux does so if there is no outgoing traffic and no incoming traffic) then the data device wants to write won't be ever picked up (it only will be picked up if host wants to send something).

At this point I would simply continue to use to workaround which is to not advertise remote wakeup (i.e. set `USB_DEVICE_REMOTE_WAKEUP` to `n`).

Reply:
Have added it back in :)

Reply:
Interesting, thanks for clarifying. Sounds like there's space for for something like a Kconfig dependency or a conditional `#warning` in the affected class drivers.

Reply:
Possibly room for improvement in the USB CDC class to add support for it also I assume